repos/fdbt-site/src/pages/api/deleteAllProducts.ts

```diff
@@ -0,0 +1,24 @@
+import { NextApiResponse } from 'next';
+import { deleteProductsByNocCode, getAllProductsByNoc } from '../../data/auroradb';
+import { redirectToError, redirectTo, getAndValidateNoc } from '../../utils/apiUtils';
+import { NextApiRequestWithSession } from '../../interfaces';
+import { deleteMultipleObjectsFromS3 } from '../../data/s3';
+import { PRODUCTS_DATA_BUCKET_NAME } from '../../constants';
+
+export default async (req: NextApiRequestWithSession, res: NextApiResponse): Promise<void> => {
+    try {
+        if (process.env.NODE_ENV === 'development' || process.env.NODE_ENV === 'test') {
```

Comment:
and in here, change this to 
```suggestion
        if (products.length > 0 && (process.env.NODE_ENV === 'development' || process.env.NODE_ENV === 'test')) {
```